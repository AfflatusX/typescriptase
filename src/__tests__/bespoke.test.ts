import { bespoke1, bespoke2, bespoke3 } from "./setup";

test(
  "bespoke1",
  async (): Promise<void> => {
    expect(bespoke1.render({
      name: "bespoke1",
      path: "src/__tests__/bespoke.test.ts",
    }))
      .toMatchSnapshot();
  },
);

test(
  "bespoke2",
  async (): Promise<void> => {
    expect(bespoke2.render({
      name: "bespoke2",
      path: "src/__tests__/bespoke.test.ts",
    }))
      .toMatchSnapshot();
  },
);

test(
  "bespoke3",
  async (): Promise<void> => {
    expect(bespoke3.render({
      name: "bespoke3",
      path: "src/__tests__/bespoke.test.ts",
    }))
      .toMatchSnapshot();
  },
);
