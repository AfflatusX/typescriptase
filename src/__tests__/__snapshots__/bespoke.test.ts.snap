// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bespoke1 1`] = `
"/* START BESPOKE SECTION <<TEST1>> */
/* END BESPOKE SECTION <<TEST1>> */
"
`;

exports[`bespoke2 1`] = `
"/* START BESPOKE SECTION <<test2>> */
/* END BESPOKE SECTION <<test2>> */
"
`;

exports[`bespoke3 1`] = `
"/* START BESPOKE SECTION <<3tESt3>> */
/* END BESPOKE SECTION <<3tESt3>> */
"
`;
